tests/detect-vlan-layers/test.rules

```diff
@@ -0,0 +1,4 @@
+alert ip any any -> any any (msg:"Packet has 3 VLAN layers"; vlan.layers:3; sid:1;)
+alert ip any any -> any any (msg:"Packet has more than 2 VLAN layers"; vlan.layers:>2; sid:2;)
+alert ip any any -> any any (msg:"The number of layers in the packet is not 1"; vlan.layers:!1; sid:3;)
+alert ip any any -> any any (msg:"Packet has 0 VLAN layers"; vlan.layers:0; sid:4;)
```

Comment:
so this means no vlan in the packet at all, right?

Can we use the keyword to test for vlan?
`vlan.layers:0` meaning no vlan in packet
`vlan.layers:>0` meaning there is a vlan header in the packet?

Reply:
> so this means no vlan in the packet at all, right?

Yes

> Can we use the keyword to test for vlan? `vlan.layers:0` meaning no vlan in packet `vlan.layers:>0` meaning there is a vlan header in the packet?

Indeed, how do you like it ?

Reply:
Makes sense to me.